hat/app/org/hatdex/hat/utils/Mailer.scala

```diff
@@ -112,6 +112,14 @@ trait HatMailer extends Mailer {
     )(implicit m: MessagesApi,
       lang: Lang,
       server: HatServer): Done
+
+  // no changes
+  def hatClaimed(
```

Comment:
The code can be made more readable by semantically aligning it with `verifyEmail` method. E.g. `emailVerified` or similar.